go/tao/linux_host.go

```diff
@@ -90,6 +91,9 @@ func NewRootLinuxHost(path string, guard Guard, password []byte, childFactory Ho
 }
 
 // LinuxHostChild holds state associated with a running child program.
+// TODO(kwalsh) Nothing in this is linux specific. Move channel and ChildSubprin
```

Comment:
Yeah, this was originally linux-specific, IIRC, but refactoring has taken that away